lib/daysOfTheWeek.js

```diff
@@ -9,3 +9,11 @@ module.exports.getDayName = (date) => {
 
   return ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'][passedDate.getDay()];
 };
+
+module.exports.next = (dow) => {
```

Comment:
This function name isn't very useful.  `next`?  Next what?  Function names should identify what they do.  Maybe this could be something like `getNextDateforDay` instead.  Also, it requires an argument called `dow`.  What is that?  Maybe it should be `dayName` or something similar, so it's clear what it is.